src/latexify/transformers/prefix_trimmer.py

```diff
@@ -0,0 +1,43 @@
+"""Transformer to trim package prefixes."""
+
+from __future__ import annotations
+
+import ast
+
+
+class PrefixTrimmer(ast.NodeTransformer):
+    """NodeTransformer to trim package prefixes.
+
+    Example:
+        import math
+
+        def foo(n):
+            return math.sqrt(n)
+
+        PrefixTrimmer(["math"]) will modify the AST of
+        the function above to below:
+
+        import math
+
+        def foo(n):
+            return sqrt(n)
+    """
+
+    def __init__(self, prefixes: list[str]):
+        """Initializer.
+
+        Args:
+            prefixes: User defined prefixes.
+        """
+        self._prefixes = prefixes
+
+    def visit_Call(self, node: ast.Call) -> ast.Name:
+        """Visitor of Call."""
+        if (
+            isinstance(node.func, ast.Attribute)
+            and node.func.value.id in self._prefixes
+        ):
+            new_node_func = ast.Name(id=node.func.attr, ctx=node.func.ctx)
+            node.func = new_node_func
+
+        return node
```

Comment:
Some comments:

1. This process does not handle nested attributes correctly, e.g., analyzing `foo.bar.baz` will produce a wrong `func`. I think we have to take either:
   - Process only the `foo.bar` syntax: check the `node.func.attr` is the exactly the `Name` node, and if not, skip processing it.
   - Support any number of attribute chains. (I think this is much desirable)
2. Avoid updating the original node, create a new `Call` node with replaced/original attributes instead. Since the original node could be a part of other ASTs, modifying it may cause unexpected side effects in other routines.
3. Every subtree (= every attribute of `Call` that inherits `ast.AST`) must be processed recursively.
4. `ctx` should be set as that of the original `Name` node, not the `Attribute`.

Reply:
I think we don't need to synthesize the new `Name`; it is enough to use the underlying `Name` in the original tree.

Reply:
> I think we don't need to synthesize the new `Name`; it is enough to use the underlying `Name` in the original tree.

Don't think I quite understood the 4th comment, neither the one I mentioned above. Could you elaborate further?

Sorry for the mess, it's my first open source contribution 😄

Reply:
Great, I think you implemented the code very quickly, and it is much clean! Sometimes I needed to point out style issues during reviewing, but in this request I could focus on the algorithm itself, which is really nice.

As for the last suggestion, it is followed by the next comment: we can re-use the original `Name`  at the end of the attribute chain in the original tree.
For example, the AST of `foo.bar` looks like:
```python
func = Attribute(
  value='foo',
  attr=Name(id='bar', ctx=something)
)
```
and the node `func.attr` holds the exact information about the new `func`. So we can simply replace the whole `func` with `func.attr`.

Reply:
In fact the ast of `foo.bar` looks like this, @odashi:
```python
>> import ast
>> print(ast.dump(ast.parse("foo.bar", mode='eval').body))

Attribute(value=Name(id='foo', ctx=Load()), attr='bar', ctx=Load())
```
And actually, from my tests with the code above, `Attribute.attr` will always be a `str`, that's why I was leaning towards instancing a new `Name` object. 

Am I missing something here?

Reply:
Just commited new code fixing some stuff we talked about on these conversations. I believe it's a bit better, but tests are still failing and I have no idea why. See if you can spot anything.

Reply:
Ah sorry I'd suggested a wrong comment. Yes you are right. The Attribute node is defined as follows:

<img width="399" alt="無題" src="https://user-images.githubusercontent.com/1023695/202090288-2416d075-e94e-467e-af65-44e649e4316d.png">